src/components/Autocomplete/components/ComboBox/ComboBox.tsx

```diff
@@ -7,7 +7,7 @@ import {PreferredPosition} from '../../../PositionedOverlay';
 import {ActionListItemDescriptor, Key} from '../../../../types';
 import {contextTypes} from '../types';
 import KeypressListener from '../../../KeypressListener';
-import {TextField} from './components';
```

Comment:
Exposing TextField and OptionList on ComboBox is really weird and we shouldn't do it, but removing it now would be a breaking change so lets just leave them there for now even though I'd bet a mars bar on nobody ever using it in that way because it's not documented.